### PR TITLE
Create ocr-key model and api

### PR DESCRIPTION
**Pull request by hellobart**

### CODE REVIEW

File: operator_ui/src/api/v2/ocrKeys.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  }
+
+  @boundMethod
+  public destroyOcrKey(id: string): Promise<jsonapi.ApiResponse<null>> {
+    return this.destroy(undefined, { keyId: id })
+  }
+
+  private create = this.api.createResource<
+    models.OcrKeyRequest,
+    presenters.OcrKey
```
Comment (DeividasK) [inline operator_ui/src/api/v2/ocrKeys.ts:54]: Why `presenters.OcrKey` and not `models.OcrKey`?

File: operator_ui/src/api/v2/ocrKeys.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+/**
+ * Destroy deletes a OcrKey.
+ *
+ * @example "<application>/off_chain_reporting_keys/:keyId"
+ */
+interface DestroyPathParams {
+  keyId: string
+}
+const DESTROY_ENDPOINT = '/v2/off_chain_reporting_keys/:keyId'
```
Comment (DeividasK) [inline operator_ui/src/api/v2/ocrKeys.ts:27]: Could we
```suggestion
const DESTROY_ENDPOINT = `${INDEX_ENDPOINT }/:keyId`
```

File: operator_ui/src/api/v2/ocrKeys.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import * as jsonapi from '@chainlink/json-api-client'
+import { boundMethod } from 'autobind-decorator'
+import * as models from 'core/store/models'
+import * as presenters from 'core/store/presenters'
+/**
+ * Create adds validates, saves a new OcrKey.
+ *
+ * @example "<application>/off_chain_reporting_keys"
+ */
+const CREATE_ENDPOINT = '/v2/off_chain_reporting_keys'
```
Comment (DeividasK) [inline operator_ui/src/api/v2/ocrKeys.ts:10]: Could we
```suggestion
const CREATE_ENDPOINT = INDEX_ENDPOINT 
```
? Or not have it all as it's redundant in RESTful APIs.

File: operator_ui/src/api/v2/ocrKeys.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
+import * as jsonapi from '@chainlink/json-api-client'
+import { boundMethod } from 'autobind-decorator'
+import * as models from 'core/store/models'
+import * as presenters from 'core/store/presenters'
+/**
+ * Create adds validates, saves a new OcrKey.
+ *
+ * @example "<application>/off_chain_reporting_keys"
```
Comment (DeividasK) [inline operator_ui/src/api/v2/ocrKeys.ts:8]:
```suggestion
 * @example "POST <application>/off_chain_reporting_keys"
```

File: operator_ui/src/api/v2/ocrKeys.ts (hunk 1 above)
Comment (hellobart) [reply operator_ui/src/api/v2/ocrKeys.ts:54]: This seems to be redundant here but i took this pattern from other types

File: operator_ui/src/api/v2/ocrKeys.ts (hunk 2 above)
Comment (hellobart) [reply operator_ui/src/api/v2/ocrKeys.ts:27]: Same as above. To simplify this i would change `INDEX_ENDPOINT` -> `ENDPOINT`.

Comment (hellobart) [reply operator_ui/src/api/v2/ocrKeys.ts:27]:
```
/**
 * Create adds validates, saves a new OcrKey.
 *
 * @example "POST <application>/off_chain_reporting_keys"
 */
const ENDPOINT = '/v2/off_chain_reporting_keys'

/**
 * Index lists OcrKeys.
 *
 * @example "GET <application>/off_chain_reporting_keys"
 */
const INDEX_ENDPOINT = ENDPOINT

/**
 * Destroy deletes a OcrKey.
 *
 * @example "DELETE <application>/off_chain_reporting_keys/:keyId"
 */
interface DestroyPathParams {
  keyId: string
}
const DESTROY_ENDPOINT = `${ENDPOINT}/:keyId`

```

Comment (DeividasK) [reply operator_ui/src/api/v2/ocrKeys.ts:27]: I like your suggestions, let's do it.

File: operator_ui/src/api/v2/ocrKeys.ts (hunk 1 above)
Comment (DeividasK) [reply operator_ui/src/api/v2/ocrKeys.ts:54]: Let's not add to `presenters` until we need to.

**Review by DeividasK — APPROVED** [review]

**Pull request merged**